### DISCUSSION TITLE
Purchased Copolit for my team; unable to utilize Copilot Chat in a JetBrains IDE.

### DISCUSSION THREAD

**Discussion by Pookiemon** [community/community#109965] [Copilot Conversations] [answered]
Purchased Copilot for my team; unable to utilize Copilot Chat in a JetBrains IDE.

The plugin is requesting me to join a waitlist

i read this document here. from my understanding," Owners of invoiced organizations or enterprises with a Copilot Business subscription who also have an account manager. To join the private beta, contact your account manager or solutions engineer to request access."

We are an invoiced org. So not sure why we don't have access.

is there a way to enable this for my org? i wish to get all my staff this (13) but we wish to test it before we go that big. 

https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide?tool=jetbrains_beta#joining-the-beta-for-github-copilot-chat-in-a-jetbrains-ide

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Pookiemon ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by davecheney** [comment, marked as answer]
@Pookiemon Chat in IDEA editors is currently a waitlisted feature. https://github.blog/changelog/2023-11-08-copilot-in-jetbrains-ides-chat-is-now-available-in-private-beta-new-welcome-guide/
